systest/Makefile

```diff
@@ -74,38 +72,125 @@ export TEMPEST_CONFIG_DIR := $(TEMPEST_VENV_DIR)/etc/tempest
 export TEMPEST_VENV_ACTIVATE := $(TEMPEST_VENV_DIR)/bin/activate
 
 # Results Directories
-export RESULTS_DIR := $(MAKEFILE_DIR)/test_results/$(PROJECT)/$(PROJECT)_$(BRANCH)
 export API_SESSION := api_$(SUBJECTCODE_ID)_$(TIMESTAMP)
 export SCENARIO_SESSION := scenario_$(SUBJECTCODE_ID)_$(TIMESTAMP)
 
+# Install the TLC application
 tlc-install:
 	cd scripts && sudo -EH ./install_tlc.sh
 
+# Run all tests, this is what buildbot calls: make functest
 functest:
-	$(MAKE) -j -C . functest_all
-
-functest_all: tempest_tests 
-
+	$(MAKE) -C . functest_all
+
+# These are all of the functests that we want to run as part of bbot job
+# We ignore the return value so that we run all of the tests
+functest_all: tlc-install tempest_tests-install
+	-$(MAKE) -C . tempest_11.5.4_overcloud
```

Comment:
Running the scenario tests in overcloud doesn't work in our TLC deployment. We can certainly let them fail, but I want to make sure we don't kill the whole job because of expected failing tests.